### DISCUSSION TITLE
DSH in Contracts

### DISCUSSION THREAD

**Discussion by vduffel99** [CMSgov/hospital-price-transparency#109] [Q&A] [answered]
Hi There - I came across a contract where reimbursement says " ((Base Rate * Weight) + DSH) is the reimbursement. I wanted to know if this would be considered an algorithm or if I can list it as gross dollars and put in an additional payor note saying "DSH is included, in addition to case rate reimbursement"?

**Comment by RidenourK** [comment, marked as answer]
@vduffel99. Thank you for your question. 

CMS is unable to comment on or provide individualized/case-by-case predeterminations as to whether a particular strategy or approach would be deemed compliant. The compliance process typically involves a comprehensive compliance review in response to a CMS audit or a complaint received through the Hospital Price Transparency website.

If the hospital’s payer-specific negotiated charge is based on an algorithm within which no standard dollar amount can be determined, then the hospital should specify what percentage or algorithm determines the dollar amount for the item or service. CMS also finalized a requirement for hospitals to display an estimated allowed amount which would provide needed context, in dollars, for instances in which the hospital’s standard charge can only be expressed as a percentage or algorithm for a specified payer’s plan. Please refer to [88 FR 82096](https://www.federalregister.gov/d/2023-24293/p-4970), #23 and the examples on this GitHub site for more information about when to encode the estimated allowed amount. 

Additionally, as indicated in the CSV and JSON documentation, the “general_contract_provisions” data element is an optional data element that can be used to encode payer contract provisions that are applicable at an aggregate level and may include variable items and services. Examples could be "stop-loss" provisions or "lesser than" provisions that apply to the claim (as opposed to each item or service on the claim). Multiple general contract provisions across payer and plan combinations can be included in this data element. If a contract provision applies to a specific item or service, use the Payer-specific Negotiated Charge: Algorithm data element and encode the Estimated Allowed Amount data element. You may provide additional context for contract provisions that apply to a specific item or service in the Additional Payer Notes data element.
